EZP-32859: Note about translating field group names (#1397)
ezsystems/developer-documentation: docs/guide/config_repository.md

user_data: User data
```

> In order to see your translations taking effect, you need to change your [Back Office language](back_office_translations.md#changing-back-office-languages) accordingly. 

Reviewer: ```suggestion
To see the results of your translations, you need to change your [Back Office language](back_office_translations.md#changing-back-office-languages) accordingly. 
```

Author: Maybe just: `To see your translations`?

Reviewer: It's okay, we can go this way.
